synapse/cortex.py

```diff
@@ -1794,6 +1796,9 @@ def getStormSvcs(self):
         return self.nexsroot.getOffset()
 
     async def getNexusOffsEvent(self, offs):
+        if not self.nexsroot.dologging:
+            raise s_exc.BadConfValu(mesg='This method not allowed without logchanges enabled')
```

Comment:
untested

Reply:
yeah this is really invalid usage. An assertion would be in order.